src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -72,8 +72,11 @@ public void doCreateConsumer(RoutingContext routingContext, JsonObject bodyAsJso
         // get the consumer group-id
         groupId = routingContext.pathParam("groupid");
 
+        // if body from HTTP request is empty (null) then set an empty JSON to use default behaviour/values
+        JsonObject requestBody = bodyAsJson != null ? bodyAsJson : new JsonObject();
```

Comment:
This shouldn't be needed now, right?

Reply:
It is in a different point in the code.
I did the change in the HttpBridge class and not here yet. See the handle method in this class where the body is got. I have to do something similar as I did in HttpBridge.

Reply:
Ah, OK then.